Sources/Runner/Commands/Runner.swift

```diff
@@ -121,25 +121,37 @@ func runDanger(logger: Logger) throws {
     // swift build; cat Fixtures/eidolon_609.json  | ./.build/debug/danger-swift
     // swiftlint:enable line_length
 
+    // This ain't optimal, but SwiftPM have _so much code_ around this.
+    // So maybe there's a better way
+    #if os(Linux)
+    let command = "which swiftc"
+    #else
+    let command = "xcrun --find swiftc"
+    #endif
+    
+    
+    let swiftCPath = try ShellOutExecutor().shellOut(command: command)
+    
     var args = [String]()
     args += ["--driver-mode=swift"] // Eval in swift mode, I think?
+    args += ["-swift-version", "5"]
+    args += bootstrapArgs()
     args += libArgs
+    args += getSwiftPMManifestArgs(swiftPath: swiftCPath)
     args += [tempDangerfilePath] // The Dangerfile
     args += Array(CommandLine.arguments.dropFirst()) // Arguments sent to Danger
     args += [dslJSONPath] // The DSL for a Dangerfile from DangerJS
     args += [dangerResponsePath] // The expected for a Dangerfile from DangerJS
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-default;
  File: Sources/Runner/Commands/Runner.swift;
  Line: 145;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)